TestFoundation/TestNSDateFormatter.swift

```diff
@@ -277,5 +278,17 @@ class TestNSDateFormatter: XCTestCase {
         XCTAssertEqual(f.string(from: testDate), "11-03-2016")
         
     }
-    
+
+    func test_setLocalizedDateFormatFromTemplate() {
+        let locale = Locale(identifier: DEFAULT_LOCALE)
+        let template = "EEEE MMMM d y hhmmss a zzzz"
+
+        let f = DateFormatter()
+        f.locale = locale
+        f.setLocalizedDateFormatFromTemplate(template)
```

Comment:
Well, here I have to test that the setLocalizedDataFormatFromTemplate will generate the same format as DateFormatter.dateFormat(fromTemplate: dateFormatTemplate, options: 0, locale: locale) if I call directly the latter how can I test it?